contrib/src/msgpack/mod.rs

```diff
@@ -0,0 +1,118 @@
+extern crate rmp_serde;
+
+use std::ops::{Deref, DerefMut};
+use std::io::Read;
+
+use rocket::outcome::Outcome;
+use rocket::request::Request;
+use rocket::data::{self, Data, FromData};
+use rocket::response::{self, Responder, content};
+use rocket::http::Status;
+
+use serde::{Serialize, Deserialize};
+
+pub use self::rmp_serde::decode::Error as MsgPackError;
+
+/// The MsgPack type: implements `FromData` and `Responder`, allowing you to easily
```

Comment:
It looks like you just replaced `JSON` with `MsgPack` here, but the wording doesn't work in some cases. Can you customize this for `MsgPack`? In particular, you should likely refer to it as `MessagePack`.